### ISSUE TITLE
Unable to install Taskboard

### ISSUE THREAD

**Issue by Zeff98** [kiswa/TaskBoard#334]
Hi Kisa ,

I'm using a Window 7 OS and running the following : 

Apache Version:2.4.17
PHP Version:5.6.16 
5.7.9 - Port defined for MySQL: 3306 

I copied the Taskboard-master folder to wamp64/www/ as per the guide 

but i can't seem to get the taskboard running..

Please advise?

Regards,
Zeff

**Comment by kiswa** [comment]
Take a look through #159 and see if that helps. You don't need MySQL, but you do need to make sure PHP is setup to allow SQLite to work.

**Comment by Zeff98** [comment]
Hi Kiswa, 

i'm not a web developer so i'm really unsure what's the correct step for the installation. i have a wamp64 server installed on my system ( Windows 7). i've refer to the case number you've given me but i checked my files is .dll and the content in the case is .so . i'm unsure if i'm on the right track honestly.

Let me list down my step that i've done and correct me if i got it wrong some where : 
1) Download from your web
2) Extract the TaskBoard-master to my wamp64/www (webserver folder)
3) Added Alias taskboard to the webserver
4) tries to access the taskboard-master local web but doesn't work

I've also tried the following :
    Copy the folders and files (you can skip the .hg* stuff) to the location you want to use TaskBoard.
    Optional: Minify everything for better performance in production. (Requires that everything but compiler.jar in build/ is executable.)
        Open the TaskBoard directory in a terminal
        Run ./build/build-all
    Open your browser to the location you put TaskBoard and log into the site with the username admin and password admin.

doesn't seem to work in windows os environment.

i'm stuck here . i would like to confirm the following : 
1)  if i need to download a window GIT-Gui to copy 
2) PHP Composer to run the php commands?

**Comment by kiswa** [comment]
I can't really help you with this, as I don't know the workings of wamp64 (I use Linux). It seems like you should be able to enable/disable php settings from it though.

As for composer, it's just to install the vendor dependencies. You can just download [the vendor files here](https://dl.dropboxusercontent.com/u/75313861/vendor.tar.gz), extract them (you'll need something like 7-zip on Windows), and copy to your TaskBoard location.

**Comment by abauzac** [comment]
Hello
I also struggled installing taskboard with Wamp too. But here is what I've done so fare : 
- git clone of repository
- `composer install ` in Terminal (I had already composer installed) : it installed some modules
- I got error 500 when trying to login. This was due to a disabled Apache extension. Simply go to your Wamp tray icon > Apache > Modules > Enable the 'expire module' and check that "rewrite module" is also enabled.
- Next I got 200 on /api/login but 401 on /boards /current... I noticed that on the 200 request to login, I got extra information with the JSON talking about deprecating feature $HTTP_RAW_POST_DATA . To avoid this message, go to your `php.ini` file (Wamp icon > php > php.ini) and find the line ; `always_populate_raw_post_data = -1` then uncomment the line (remove the coma at the beginning). 

Now it works for me. 
Hope it helps
